saga-spring/src/test/java/io/servicecomb/saga/spring/SagaControllerTest.java

```diff
@@ -75,4 +88,84 @@ public void retrievesAllEvents() throws Exception {
         .andExpect(status().isOk())
         .andExpect(content().json(objectMapper.writeValueAsString(events)));
   }
+
+  @Test
+  public void queryRequests() throws Exception {
+    long id = 0L;
+
+    String sagaId1 = "xxxx";
+    SagaEventEntity saga1StartEvent = new SagaEventEntity(++id, sagaId1, new Date().getTime(),
+        SagaStartedEvent.class.getSimpleName(), "{}");
+    Thread.sleep(50);
```

Comment:
I don't think we need to use the sleep that way, as we can simple  add the time for it.

Reply:
Fixed